src/modules/layout/components/header/user-menu.tsx

```diff
@@ -18,7 +19,7 @@ export function UserMenu() {
 
   const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
     setAnchorEl(event.currentTarget);
-    setOpen((prev) => !prev);
+    setOpen(() => !open);
```

Comment:
Old implementation is more reliable. Just `setOpen(!open)` works too. Not an issue tho.